app/controllers/shelters_controller.rb

```diff
@@ -43,7 +43,7 @@ def destroy
       redirect_to '/shelters'
     else
       flash[:notice] = "Could not delete shelter, shelter has pending adoptions"
-      redirect_to "/shelters/#{params[:id]}"
+      redirect_back fallback_location: "/shelters"
```

Comment:
needed this, for usability, will redirect you back to the page you are on when you click delete shelter link